### ISSUE TITLE
qTox 1.16.3 on Windows 7 pro - contact list jitters based on user activity

### ISSUE THREAD

**Issue by byrnejb** [qTox/qTox#6055]
##### Brief Description

When the user interface has `By Name|All ` chosen the contact list nonetheless jitters  based on the connection status of the contacts without regard to the alphabetic order.

OS: Windows 7 Pro
qTox version: 1.16.3  
Commit hash:  8eed684c37bd711de2ca4bd863f5f0509edcdc2d
toxcore:  0.2.3
Qt:  ?
…

Reproducible: Always 

##### Steps to reproduce

1. Start qTox
2. load profile with mutiple contacts
3. Set contact list sort order to By Name and select ALL for names to display
4. Wait and watch as connections are made and the most reent is always moved to the top of the lsit

##### Observed Behavior
Names in contact list change positions based on most recent connection status

##### Expected Behavior
Names remain fixed in place based on aphabetic sort order and only the connection status changes

##### Additional Info

none

**Comment by anthonybilinski** [comment]
Even with "By Name" selected, the contact list still shows groups, online friends, and offline friends as three different sorted groups. Is that the behaviour you're seeing, e.g. friends moving from the correct location in the "offline" group to the correct location in the "online" group when you connect to them, or is there additional jittering you're seeing?

**Comment by byrnejb** [comment]
That may be it.  I cannot use qTox on my work desktop (FreeBSD-12.1) because it segfaults.  I need to check this with my Mac or with one of the user's MS desktops before I can confirm this.  

In any case, if this is the behaviour then it is very distracting when there is any sort of network problem at either end.  In those situations the online/offline status can be toggled multiple time is a short period.  If there was an option to just display the list of contacts the desired order and only have the status indicator update it would be a lot easier on the eyes.

This is not a theoretical problem. We are using qTox to manage our remote workers during the lockdown/quarantine  and we have noted a constant changing of status for some of them.  This is not caused by their shutting down the program or by being inactive.  It seems related to the quality of their network connections at home.  But for the rest of us it makes the contact list look as if it has fleas or is throwing a fit.

**Comment by anthonybilinski** [comment]
>  I cannot use qTox on my work desktop (FreeBSD-12.1) because it segfaults.

Yikes, is that related to #6054? If not, if you have repro steps or a backtrace from this please open another issue.

> If there was an option to just display the list of contacts the desired order and only have the status indicator update it would be a lot easier on the eyes.

Understood. I'd like to improve the activity sorting, and when that's done the alphabetical sorting could be more strictly alphabetical instead of the current kind of combination of activity + alphabetical.

**Comment by byrnejb** [comment]
> Yikes, is that related to #6054? If not, if you have repro steps or a backtrace from this please open another issue.

No.  qTox runs on Windows 7 and then it stops working.  I believe that the issue may be tied to where the user's profile is kept.  I am going to test that hypothesis this evening.

As for the segfault.  I will try and obtain what you desire tomorrow.  I am not familiar with the debugging tools on FreeBSD but I am confident that I will puzzle them out.
